### PR TITLE
[main] add config for vsphere in jenkins, with updates.

### PR DESCRIPTION
**Pull request by slickwarren**
## Issue: 

partly solves https://github.com/rancher/qa-tasks/issues/1617
 
## Problem

 we currently have no way to run our tests on our infrastructure. 

## Solution

 use jenkins worker behind the VPN in order to run specific jobs. 
   * this should give the jenkins job access to both vsphere and harvester envs. 

also updated some default branches while I'm in the code. 
This will require a jjb PR before it can be used. 

## Testing

tested in jenkins

### CODE REVIEW

**Review by markusewalker — APPROVED** [review]
LGTM. This will need backports as well for 2.9 and 2.8.

File: tests/v2/validation/Jenkinsfile.vsphere [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+    }
+    wrap([$class: 'AnsiColorBuildWrapper', 'colorMapName': 'XTerm', 'defaultFg': 2, 'defaultBg':1]) {
+      withFolderProperties {
+        paramsMap = []
+        params.each {
+          if (it.value && it.value.trim() != "") {
+              paramsMap << "$it.key=$it.value"
+          }
+        }
+        withCredentials([ string(credentialsId: 'AWS_ACCESS_KEY_ID', variable: 'AWS_ACCESS_KEY_ID'),
```
[the remark this answers is not in the document]
Comment (slickwarren) [reply tests/v2/validation/Jenkinsfile.vsphere:36]: yes, the creds will be in the config itself. We should still load in all credentials, as you will still be able to run aws from this node. Its just that you can't run vsphere from the aws nodes, if that makes sense. This is why we need to specify the node for now in order to run vsphere / harvester / stuff behind the vpn.

File: tests/v2/validation/Jenkinsfile.vsphere [hunk 2]
```diff
@@ -0,0 +1,148 @@
+#!groovy
+node("vsphere-vpn-1"47) {
```
[the remark this answers is not in the document]
Comment (slickwarren) [reply tests/v2/validation/Jenkinsfile.vsphere]: resolved.

**Review by lscalabrini01 — APPROVED** [review]
LGTM

**Review by anupama2501 — APPROVED** [review]

**Pull request merged**